src/__mocks__/news-rss.ts

```diff
@@ -0,0 +1 @@
//media.defense.gov/2021/May/19/2002664619/82/55/0/210519-F-GO452-0001.JPG\' alt=\'Department of the Air Force News\' /&gt; &lt;br /&gt;\r\n\r\n</description>\r\n      <pubDate>Fri, 04 Jun 2021 18:57:51 GMT</pubDate>\r\n      <dc:creator>DoD News</dc:creator>\r\n      <guid isPermaLink="false">https://www.spaceforce.mil/News/Article/2646698/department-of-the-air-force-announces-fourth-vanguard-program/</guid>\r\n      <enclosure url="https://media.defense.gov/2021/May/19/2002664619/670/394/0/210519-F-GO452-0001.JPG" type="image/jpeg" />\r\n    </item>\r\n    <item>\r\n      <title>DAF releases FY22 budget proposal, journeys to the Air and Space Forces of 2030  Secretary of the Air Force Public Affairs</title>\r\n      <link>https://www.spaceforce.mil/News/Article/2639108/daf-releases-fy22-budget-proposal-journeys-to-the-air-and-space-forces-of-2030/</link>\r\n      <description>The Department of the Air Force released its Fiscal Year 2022 budget proposal, May 28, focusing on investing in people and capability, building the future force and delivering joint lethality and effectiveness.&lt;br/&gt; \r\n&lt;img src=\'https://media.defense.gov/2021/May/19/2002664619/82/55/0/210519-F-GO452-0001.JPG\' alt=\'Department of the Air Force News\' /&gt; &lt;br /&gt;\r\n\r\n</description>\r\n      <pubDate>Fri, 28 May 2021 20:30:48 GMT</pubDate>\r\n      <dc:creator>DoD News</dc:creator>\r\n      <guid isPermaLink="false">https://www.spaceforce.mil/News/Article/2639108/daf-releases-fy22-budget-proposal-journeys-to-the-air-and-space-forces-of-2030/</guid>\r\n      <enclosure url="https://media.defense.gov/2021/May/19/2002664619/670/394/0/210519-F-GO452-0001.JPG" type="image/jpeg" />\r\n    </item>\r\n    <item>\r\n      <title>Senate committee considers Frank Kendall’s nomination to be the 26th Air Force secretary</title>\r\n      <link>https://www.spaceforce.mil/News/Article/2632370/senate-committee-considers-frank-kendalls-nomination-to-be-the-26th-air-force-s/</link>\r\n      <description>Frank Kendall told the Senate Armed Services Committee May 25 that if he is confirmed as the 26th Secretary of the Air Force he will continue reshaping the Air and Space forces to confront near-peer adversaries such as China, nurture the Space Force’s successful evolution and ensure that Airmen, Guardians and their families are well served.&lt;br/&gt; \r\n&lt;img src=\'https://media.defense.gov/2021/May/19/2002664619/82/55/0/210519-F-GO452-0001.JPG\' alt=\'Department of the Air Force News\' /&gt; &lt;br /&gt;\r\n\r\n</description>\r\n      <pubDate>Tue, 25 May 2021 21:35:42 GMT</pubDate>\r\n      <dc:creator>Mr. Charles Pope</dc:creator>\r\n      <guid isPermaLink="false">https://www.spaceforce.mil/News/Article/2632370/senate-committee-considers-frank-kendalls-nomination-to-be-the-26th-air-force-s/</guid>\r\n      <enclosure url="https://media.defense.gov/2021/May/19/2002664619/670/394/0/210519-F-GO452-0001.JPG" type="image/jpeg" />\r\n    </item>\r\n  </channel>\r\n</rss>'
```

Comment:
Very small thing, but the & in the title "FY22 budget gives Air &amp; Space Forces the strength to meet threats, Roth, Brown, Raymond tell Senate" isn't rendering in the browser

Reply:
👍  good catch!! That should be fixed now, along with 💯  test coverage 😂